### PR TITLE
Update return type declaration for getGeneral function

### PR DESCRIPTION
**Pull request by DariusIII**
Current return type declaration causes exception when function is used.

### CODE REVIEW

**Comment by Fossil01** [comment]
Tested and it works.

File: src/Container/MediaInfoContainer.php [hunk 1]
```diff
@@ -65,9 +65,9 @@ class MediaInfoContainer implements \JsonSerializable
     private $others = [];
 
     /**
-     * @return General
+     * @return General|null
      */
-    public function getGeneral(): \Mhor\MediaInfo\Type\General
+    public function getGeneral(): ?object
```
Comment (mhor) [inline src/Container/MediaInfoContainer.php:70]: Could you specify object return  type:
```suggestion
    public function getGeneral(): ?General
```

**Comment by mhor** [comment]
Thanks @DariusIII, Merged and released as [5.0.1](https://github.com/mhor/php-mediainfo/releases/tag/5.0.1)

**Comment by DariusIII** [comment]
Thank you too, @mhor !

**Pull request merged**